packages/calypso-build/webpack/util.js

```diff
@@ -85,9 +85,12 @@ const nodeModulesToTranspile = [
 	'gridicons/',
 	'prismjs/',
 	'punycode/',
+	'query-string/',
 	'react-spring/',
 	'regenerate-unicode-properties/',
 	'regexpu-core/',
+	'split-on-first',
+	'strict-uri-encode',
```

Comment:
These two are missing the trailing `/`
```suggestion
	'split-on-first/',
	'strict-uri-encode/',
```

Reply:
Good point. Done in a27f05454ac12e19edc507bfe010ef06948157be